doc/src/conf.py

```diff
@@ -142,7 +142,52 @@
 html_last_updated_fmt = '%b %d, %Y'
 
 # was classic
-html_theme = "classic"
+# html_theme = "classic"
+html_theme = "furo"
+
+css_settings_common = {
+    "color-brand-primary": "#81B953",
+    "color-brand-content": "#29A329",
+    "color-sidebar-background": "#3B5526",
+    "color-sidebar-link-text": "#FFFFFF",
+    "color-sidebar-brand-text": "#DDDDDD",
+    "color-sidebar-link-text--top-level": "#FFFFFF",
+    "color-sidebar-search-icon": "#DDDDDD",
+    "color-sidebar-item-background--hover": "#8AB950",
+    "color-sidebar-item-expander-background--hover": "#8AB950",
+
+        "color-api-keyword": "#29A329",
+    "color-foreground-secondary": "#53555B",
+    "color-link-underline--hover": "#29A329",
+
+    # TODO: Add the other types of admonitions here if anyone uses them.
+    "color-admonition-title-background--seealso": "var(--color-brand-primary)",
+    "color-admonition-title--seealso": "black",
+    "color-admonition-title-background--note": "var(--color-brand-primary)",
+    "color-admonition-title--note": "black",
+    "color-admonition-title-background--warning": "var(--color-problematic)",
+    "color-admonition-title--warning": "white",
+
+    # Note: this doesn't work. If we want to change this, we have to set
+    # it as the .highlight background in custom.css.
+    "color-code-background": "hsl(80deg 100% 95%)",
+
+    "admonition-font-size": "var(--font-size--normal)",
+    "admonition-title-font-size": "var(--font-size--normal)",
+    "code-font-size": "var(--font-size--small)",
+    "font-stack--monospace": 'DejaVu Sans Mono,"SFMono-Regular",Menlo,Consolas,Monaco,Liberation Mono,Lucida Console,monospace;'
+}
+
+html_theme_options = {
+    "light_css_variables": css_settings_common,
+    "dark_css_variables": {**css_settings_common,
```

Comment:
You shouldn't need this -- the light mode variables get inherited for dark mode.

Reply:
See https://pradyunsg.me/furo/customisation/colors/#how-light-and-dark-mode-work for the relevant details.

Reply:
I didn't realize this. It would be nice if this were mentioned at https://pradyunsg.me/furo/customisation/#light-css-variables-dark-css-variables

Reply:
This was working, but I updated to the latest Furo and it stopped working and I had to separate them again.